website/addons/base/__init__.py

```diff
@@ -69,7 +69,7 @@ def __init__(self, short_name, full_name, owners, categories,
                  node_settings_model=None, user_settings_model=None, include_js=None, include_css=None,
                  widget_help=None, views=None, configs=None, models=None,
                  has_hgrid_files=False, get_hgrid_data=None, max_file_size=None, high_max_file_size=None,
-                 accept_extensions=True,
+                 accept_extensions=True, description='', url=None,
```

Comment:
The configuration object *is* the addon. It holds all the meta-information about the addon, such as name, category (`'storage'`, `'security'`, `'documentation'`), associated models, etc.

As for :camel: straws, I agree that the quantity of `args` is pushing absurdity, but that refactor seemed more like an `Improvement` than a `Feature`.